src/templates/en/2019/base.html

```diff
@@ -154,7 +154,10 @@
     "cdn": "CDN",
     "page-weight": "Page Weight",
     "resource-hints": "Resource Hints",
-    "http2": "HTTP/2"
+    "http2": "HTTP/2",
+    "privacy": "Privacy",
+    "capabilities": "Capabilities",
+    "jamstack": "JAMstack"
```

Comment:
@ahmadawais is this the correct capitalization?

Reply:
Yeah there seems some difference of opinion here:

Jamstack:
https://jamstack.org/
https://www.netlify.com/jamstack/

JAMstack:
https://jamstack.wtf/

Would be good to get agreement on what we are going to use.

Reply:
Since you ask, as the term has been coined by Netlify, I would go with them on Jamstack, but I don't think it really matters.

Reply:
Apparently the official word is Jamstack: https://github.com/jamstack/jamstack.org/issues/279

Will change this PR for not but feel free to disagree before it's merged! @rviscomi / @ahmadawais we should go through and update all other references in the repo if we're in agreement on this.

Reply:
Used to be JAMstack but after a healthy discussion and lots of confusion it's now Jamstack. While I don't like the later I think that's the way to go. So go ahead with Jamstack.